build.gradle

```diff
@@ -448,22 +449,13 @@ task github(dependsOn: (signingEnabled ? signRemapJar : remapJar)) {
 	}
 }
 
-task modrinth(type: com.modrinth.minotaur.TaskModrinthUpload, dependsOn: (signingEnabled ? signRemapJar : remapJar)) {
-	onlyIf {
-		ENV.MODRINTH_TOKEN
-	}
-
-	token = ENV.MODRINTH_TOKEN
```

Comment:
Where does the token get passed in now?

Reply:
`$MODRINTH_TOKEN` is the default: <https://github.com/modrinth/minotaur/blob/b27fc1b/src/main/java/com/modrinth/minotaur/ModrinthExtension.java#L27>
If you'd like I can change it to explicitly use it in the buildscript.